packages/api/src/rich-text/util.ts

```diff
@@ -0,0 +1,6 @@
+export const MENTION_REGEX = /(^|\s|\()(@)([a-zA-Z0-9.-]+)(\b)/g
+export const URL_REGEX =
+  /(^|\s|\()((https?:\/\/[\S]+)|((?<domain>[a-z][a-z0-9]*(\.[a-z0-9]+)+)[\S]*))/gim
+export const TAG_REGEX =
+  /(^|\s)#((?!\ufe0f)[^\s\u00AD\u2060\u200A\u200B\u200C\u200D]*)?/g
```

Comment:
```suggestion
  /(^|\s)#((?!\ufe0f|\u20e2)[^\s\u00AD\u2060\u200A\u200B\u200C\u200D]*)?/g
```

This prevents matching `#⃢test` as a hashtag

Reply:
🙏 you da best